packages/platform-test-suite/test/functional/dapi/subscribeToBlockHeadersWithChainLocksHandlerFactory.spec.js

```diff
@@ -0,0 +1,177 @@
+const DAPIClient = require('@dashevo/dapi-client');
+const {
+  Block,
+  BlockHeader,
+  PrivateKey,
+} = require('@dashevo/dashcore-lib');
+
+const getDAPISeeds = require('../../../lib/test/getDAPISeeds');
+
+const wait = (ms) => new Promise((resolve) => {
+  setTimeout(resolve, ms);
+});
+
+describe('subscribeToBlockHeadersWithChainLocksHandlerFactory', () => {
```

Comment:
This test will take eternity on testnet. Should we limit the number of historical blocks we are getting?

Reply:
It is limited to 10 blocks in every test case. Is this too much?

Reply:
Oh... I missed that. That exact number what I had in my mind ))